error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -106,7 +106,7 @@ Mono<T> after(T value) {
     }
   }
 
-  /** Prefer {@link Mono#justOrEmpty(Optional)} over more verbose alternatives. */
+  /** Prefer {@code () -> Mono#justOrEmpty(Optional)} over more verbose alternatives. */
```

Comment:
👍 ? 👎 ? 🤷 ?

Reply:
Hmm, instead we should mention `Mono.defer`. (And rename the rule...)